app/models/well_attribute.rb

```diff
@@ -74,4 +76,13 @@ def current_volume=(current_volume)
     current_volume = 0.0 if current_volume.to_f < 0
     super
   end
+
+  def broadcast_warehouse_message
+    message = Messenger.find_by(target_id: well_id)
+    if message.present?
+      message.resend
+    else
+      Messenger.new(target: self, template: 'WellStockResourceIO', root: 'stock_resource').broadcast
+    end
```

Comment:
This is changing current behaviour. I think is not right because is treating every well as if it was a stock well, but we may be updating other wells that are not stock and end up sending messages from them as stock wells.